conans/client/conf/config_installer.py

```diff
@@ -202,18 +207,13 @@ def _handle_hooks(src_hooks_path, dst_hooks_path, output):
     :param dst_hooks_path:  Folder where the hooks should finally go
     :param output: Output to indicate the files copied
     """
-    hooks_dirs = []
     for root, dirs, files in walk(src_hooks_path):
-        if root == src_hooks_path:
-            hooks_dirs = dirs
-        else:
-            copied_files = False
-            relpath = os.path.relpath(root, src_hooks_path)
-            for f in files:
-                if ".git" not in f:
-                    dst = os.path.join(dst_hooks_path, relpath)
-                    mkdir(dst)
-                    shutil.copy(os.path.join(root, f), dst)
-                    copied_files = True
-            if copied_files and relpath in hooks_dirs:
+        if ".git" in root:
```

Comment:
are we going to check for explicit `.git` folder, or just something containing `.git` (like `.gitattributes`, `.gitignore` or `my_repository.git`)?

Reply:
both cases, if a dir or file contains `.git`, it will be ignored. do you see some corner case?

Reply:
maybe we should use == ".git" instead

Reply:
but as you commented before, .gitattributes and .gitignore will be copied.